src/CallbackRegistrar.php

```diff
@@ -141,4 +149,49 @@ public function hasProxy(string $proxy)
     {
         return array_key_exists($proxy, $this->proxies);
     }
+
+    /**
+     * Set the default callback and its arguments.
+     *
+     * @param string|callable $callback
+     * @param  mixed ...$arguments
+     *
+     * @return void
+     */
+    public function setDefault($callback, ...$arguments)
+    {
+        if (is_callable($callback)) {
+            return $this->default = [$callback, $arguments];
```

Comment:
According to the doc block `@return void` but this clearly returns the default.

Reply:
Oh that is a typo, forgot to change the auto generated return type, i will also add the @throws exception